[FP]: CVE-2022-31569 jakarta.annotation-api@1.3.5 & javax.jms-api-2.0

### Package URl

pkg:maven/jakarta.annotation/jakarta.annotation-api@1.3.5, pkg:maven/javax.jms/javax.jms-api@2.0

### CPE

`cpe:2.3:a:oracle:java_se:1.3.5:*:*:*:*:*:*:*, cpe:2.3:a:oracle:projects:1.3.5:*:*:*:*:*:*:*, cpe:2.3:a:projects_project:projects:1.3.5:*:*:*:*:*:*:*`

### CVE

CVE-2022-31569

### ODC Integration

{"label"=>"Gradle Plugin"}

### ODC Version

7.1.1

### Description

overly inclusive CPE?
https://nvd.nist.gov/vuln/detail/CVE-2022-31569
https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-31569

We hit the same thing (including CVE-2022-31548 ) 
```
[ERROR] jakarta.annotation-api-1.3.5.jar: CVE-2022-31569(9.3)
[ERROR] jakarta.transaction-api-1.3.3.jar: CVE-2022-31569(9.3)
[ERROR] querydsl-core-5.0.0.jar: CVE-2022-31548(9.3)
[ERROR] spring-cloud-commons-3.1.3.jar: CVE-2022-31569(9.3)
[ERROR] spring-security-rsa-1.0.10.RELEASE.jar: CVE-2022-31569(9.3)
```
These CVE descriptions both seem to reference github repositories
> The RipudamanKaushikDal/projects repository through 2022-04-03 on GitHub allows absolute path traversal because the Flask send_file function is used unsafely. 

and
> The nrlakin/homepage repository through 2017-03-06 on GitHub allows absolute path traversal because the Flask send_file function is used unsafely.

Has anyone else noticed the quality of the CVEs has gone down recently? There was a CVE for a documentation bug recently (in some example code!). This seems a bit extreme and if like us it breaks the build then it generates a lot of effort.

--- a/core/src/main/resources/dependencycheck-base-suppression.xml
+++ b/core/src/main/resources/dependencycheck-base-suppression.xml
@@ -5290,5 +5290,71 @@
         <packageUrl regex="true">^pkg:maven/(?!org\.springframework/spring\-web@).*$</packageUrl>
         <cve>CVE-2016-1000027</cve>
     </suppress>
+    <suppress base="true">
+        <notes><![CDATA[
+   FP per issue #4671, CVEs for Python pet projects
+   ]]></notes>
+        <packageUrl regex="true">^.*$</packageUrl>
+        <cve>CVE-2022-31504</cve>
+        <cve>CVE-2022-31505</cve>
+        <cve>CVE-2022-31509</cve>
+        <cve>CVE-2022-31510</cve>
+        <cve>CVE-2022-31511</cve>
+        <cve>CVE-2022-31512</cve>
+        <cve>CVE-2022-31513</cve>
+        <cve>CVE-2022-31514</cve>
+        <cve>CVE-2022-31515</cve>
+        <cve>CVE-2022-31516</cve>
+        <cve>CVE-2022-31518</cve>
+        <cve>CVE-2022-31520</cve>
+        <cve>CVE-2022-31521</cve>
+        <cve>CVE-2022-31526</cve>
+        <cve>CVE-2022-31527</cve>
+        <cve>CVE-2022-31528</cve>
+        <cve>CVE-2022-31532</cve>
+        <cve>CVE-2022-31533</cve>
+        <cve>CVE-2022-31534</cve>
+        <cve>CVE-2022-31535</cve>
+        <cve>CVE-2022-31536</cve>
+        <cve>CVE-2022-31537</cve>
+        <cve>CVE-2022-31538</cve>
+        <cve>CVE-2022-31540</cve>
+        <cve>CVE-2022-31544</cve>
+        <cve>CVE-2022-31545</cve>
+        <cve>CVE-2022-31546</cve>
+        <cve>CVE-2022-31547</cve>
+        <cve>CVE-2022-31548</cve>
+        <cve>CVE-2022-31551</cve>
+        <cve>CVE-2022-31552</cve>
+        <cve>CVE-2022-31553</cve>
+        <cve>CVE-2022-31554</cve>
+        <cve>CVE-2022-31555</cve>
+        <cve>CVE-2022-31556</cve>
+        <cve>CVE-2022-31557</cve>
+        <cve>CVE-2022-31559</cve>
+        <cve>CVE-2022-31560</cve>
+        <cve>CVE-2022-31561</cve>
+        <cve>CVE-2022-31562</cve>
+        <cve>CVE-2022-31563</cve>
+        <cve>CVE-2022-31564</cve>
+        <cve>CVE-2022-31565</cve>
+        <cve>CVE-2022-31566</cve>
+        <cve>CVE-2022-31567</cve>
+        <cve>CVE-2022-31568</cve>
+        <cve>CVE-2022-31570</cve>
+        <cve>CVE-2022-31571</cve>
+        <cve>CVE-2022-31572</cve>
+        <cve>CVE-2022-31574</cve>
+        <cve>CVE-2022-31575</cve>
+        <cve>CVE-2022-31576</cve>
+        <cve>CVE-2022-31577</cve>
+        <cve>CVE-2022-31578</cve>
+        <cve>CVE-2022-31579</cve>
+        <cve>CVE-2022-31582</cve>
+        <cve>CVE-2022-31583</cve>
+        <cve>CVE-2022-31585</cve>
+        <cve>CVE-2022-31587</cve>
+        <cve>CVE-2022-31588</cve>
+    </suppress>
     <!-- endregion -->
 </suppressions>

